README.md

```diff
@@ -37,15 +48,18 @@ $ curl \
     http://localhost:5000/predictjson
 ```
 
+### Output
+The output is a scalar value: `0` or `1`, which corresponds to whether or not a student is predicted to be successful by the end of the year, as measured by their final grade. A value of `1` means that a student is predicted to be successful whereas `0` means otherwise.
+
 ## Model Description
 
 ### Input Features
 The features we used in training our model are `health`, `absences`, `age`, and `G2`.
 
 ### Improvement from the Baseline Model
-By taking in G2 as an additional feature, our retrained model achieves an f1 score of 0.9861111111111112 on the test set. In comparison, the f1 score of the baseline model on the test set is 0.5185185185185185.
+By taking in G2 as an additional feature, our retrained model achieves an f1 score of `0.986 `on the test set. In comparison, the f1 score of the baseline model on the test set is `0.519`.
 
 Thus, our retrained model has a significantly better out-of-sample accuracy than the baseline model, which indicates better model quality.
 
 ## Testing
-**(what testing have we done? How do we run tests?)**
+**(what testing have we done? How do we run tests?)**
```

Comment:
tests can be run by running pytest in the parent directory. They require that the container be running. (I can add the testing info if you want).

Reply:
If you could add this, that would be great. Since you wrote the tests, I think that you'd probably know how to best explain this part